internal/pkg/cli/env_show.go

```diff
@@ -0,0 +1,341 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+package cli
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"sort"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/describe"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/color"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
+	"github.com/spf13/cobra"
+)
+
+const (
+	environmentShowProjectNamePrompt     = "Which project's environment would you like to show?"
+	environmentShowProjectNameHelpPrompt = "A project groups all of your applications together."
+	environmentShowEnvNamePrompt         = "Which environment of %s would you like to show?"
+	environmentShowEnvNameHelpPrompt     = "The detail of an environment will be shown (e.g., Account ID, Apps, Tags)."
+)
+
+type showEnvVars struct {
+	*GlobalOpts
+	shouldOutputJSON      bool
+	shouldOutputResources bool
+	envName               string
+}
+
+type showEnvOpts struct {
+	showEnvVars
+
+	w             io.Writer
+	storeSvc      storeReader
+	describer     webAppDescriber
+	// Note: ws will be removed; put it back in for now so code would compile.
+	ws 				wsAppReader
+	initDescriber func(*showEnvOpts) error // Overriden in tests.
+}
+
+func newShowEnvOpts(vars showEnvVars) (*showEnvOpts, error) {
+	ssmStore, err := store.New()
+	if err != nil {
+		return nil, fmt.Errorf("connect to environment datastore: %w", err)
+	}
+	//ws, err := workspace.New()
+	//if err != nil {
+	//	return nil, err
+	//}
+
+	return &showEnvOpts{
+		showEnvVars: 	vars,
+		storeSvc:       ssmStore,
+		w:              log.OutputWriter,
+		initDescriber: func(o *showEnvOpts) error {
+			d, err := describe.NewWebAppDescriber(o.ProjectName(), o.envName)
+			if err != nil {
+				return fmt.Errorf("creating describer for environment %s in project %s: %w", o.envName, o.ProjectName(), err)
+			}
+			o.describer = d
+			return nil
```

Comment:
We probably don't need the `WebAppDescriber` as that fetches information for an application. We'd need to implement something similar for an environment, like an `EnvDescriber`.